tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`5-10 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (with attributes) using Local Fabric CA.
```

Comment:
i don't think you want the `with attributes` part in brackets

Reply:
Fixed